examples/amp-story/bookendv2.json

```diff
@@ -0,0 +1,20 @@
+{
+  "bookend-version": "v2.0",
+  "share-providers": [
```

Comment:
@newmuis not sure what you think of this interface. It looks cleaner to me to just write the providers name, and in the case of facebook make it an object with the app_id as the value. Downside is that maybe it's not as verbose as writing `{"provider": "facebook", "app_id": "1235"}`. WDYT?

Reply:
There are other options that one may or may not need to set, apart from the app id;. This format loses the ability to specify more than one parameter per provider.